BMI160.h

```diff
@@ -595,7 +799,9 @@ class BMI160Class {
         bool getIntDataReadyStatus();
 
         void getMotion6(int16_t* ax, int16_t* ay, int16_t* az, int16_t* gx, int16_t* gy, int16_t* gz);
-        void getAcceleration(int16_t* x, int16_t* y, int16_t* z);
+        void getMotion9(int16_t* ax, int16_t* ay, int16_t* az, int16_t* gx, int16_t* gy, int16_t* gz, int16_t* mx, int16_t* my, int16_t* mz, uint16_t* rh);		//Added for BMM150 Support
+		void extractMotion9(uint8_t* buffer, int16_t* ax, int16_t* ay, int16_t* az, int16_t* gx, int16_t* gy, int16_t* gz, int16_t* mx, int16_t* my, int16_t* mz, uint16_t* rh);		//Added for BMM150 Support
+		void getAcceleration(int16_t* x, int16_t* y, int16_t* z);
```

Comment:
Could you please eliminate these unintentional changes?
Please fix indentation here.